components/browser/toolbar/src/main/java/mozilla/components/browser/toolbar/BrowserToolbar.kt

```diff
@@ -323,6 +323,13 @@ class BrowserToolbar @JvmOverloads constructor(
         edit.addEditActionEnd(action)
     }
 
+    /**
+     * Hide end actions in edit mode.
+     */
+    override fun showEditActionEnd(isVisible: Boolean) {
```

Comment:
I am wondering if there is a way to just observe that the text is empty and re-show the actions instead of explicitly calling to toggle its visibility?

Reply:
Sure, I'll make this auto detect and change the API so the app can decides if they want to use the feature.  This is mostly because currently this feature is for unified search only.